skfda/ml/classification/_logistic_regression.py

```diff
@@ -91,7 +92,8 @@ def fit(  # noqa: D102
 
         selected_indexes = np.zeros(self.p, dtype=np.intc)
 
-        mvlr = mvLogisticRegression()  # multivariate logistic regression
+        # multivariate logistic regression
+        mvlr = mvLogisticRegression(penalty='l2')
```

Comment:
Does the original algorithm use L2 penalization?

Reply:
I don´t think so, but the original algorithm use firth's estimator, which we don´t use. I have tried to use no penalty but it fails (apparently there is a division by zero somewhere). Besides, in this algorithm, I don´t think it really matters which penalty we use in the multivariate part.